### PR TITLE
Add Traditional Chinese README file

### PR DESCRIPTION
**Pull request by Minato1123**
Refer to the README contents in Japanese and other languages, and use ChatGPT and Google Translate to assist with translation and editing.

### CODE REVIEW

File: README_TC.md
Comment (allen870619) [inline README_TC.md:1]: It might be better if rename to `README_zhHant` for file naming alignment.

File: README_TC.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+# KawaiiLogos
+
+您好，晚安。這個儲存庫中上傳的是由 Sawaratsuki（さわらつき）所創作的 Logo。
+
+> [!WARNING]  
+ 這裡的 Logo 皆由 Sawaratsuki 所創作，並非各個服務或組織自行創作。  
+ 並非全部都是官方使用的 Logo。請注意過度攝取 kawaii（可愛）元素。  
+
+> [!IMPORTANT]  
+  此儲存庫的內容不得用於任何 AI 或 Sawaratsuki 認為是 AI 的設備。
```
Comment (allen870619) [inline README_TC.md:10]: 這邊寫作設備好像有點怪怪的，比較是禁止 AI 相關的行為（包含訓練、製圖等）
```suggestion
  此儲存庫的內容不得用於任何與 AI 或 Sawaratsuki 認定是 AI 的相關行為。
```

File: README_TC.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
+# KawaiiLogos
+
+您好，晚安。這個儲存庫中上傳的是由 Sawaratsuki（さわらつき）所創作的 Logo。
+
+> [!WARNING]  
+ 這裡的 Logo 皆由 Sawaratsuki 所創作，並非各個服務或組織自行創作。  
+ 並非全部都是官方使用的 Logo。請注意過度攝取 kawaii（可愛）元素。
```
Comment (allen870619) [inline README_TC.md:7]: 後面原文的意思是偏向 `請小心不要攝取過度的 kawaii 元素。`
```suggestion
 並非全部都是官方使用的 Logo。請小心不要攝取過度的 kawaii 元素。  
```

**Comment by allen870619** [comment]
Nice work! I've made some suggestions for this pull request. ☺️
Maybe can improve the translation.

**Comment by Minato1123** [comment]
@allen870619 Thank you for your suggestion! I think it's great. 🙌🏻

**Comment by T-taku** [comment]
Thank you for your PR!
However, it seems happened conflict on this.
I'd like you to fix them🙇

**Pull request closed without merging**